src/launcher/actions/appsActions.js

```diff
@@ -337,6 +338,7 @@ export function loadOfficialApps(appName, appSource) {
 
 export function installOfficialApp(name, source) {
     return dispatch => {
+        dispatch(sendLauncherUserData(`Install ${name}`, source));
```

Comment:
It'd be a better idea to have 3 arguments for `sendLauncherUserData(action, name, source)` and use `EventAction.INSTALL_APP` instead of "Install".

Reply:
Updated.